frontend/src/components/ContentPanel/hooks.ts

```diff
@@ -174,8 +175,11 @@ export const usePanel = (id: string) => {
 
     /**
      * Saves the panel state
+     * Does not save the state if needsSaving is false unless 'force' is true
      */
-    const saveState = async () => {
```

Comment:
force is currently unused, even though it SHOULD be used by the "save" button in the panel dropdown menu (cogwheel)